src/main/resources/db/migration/V1_4__update_timestamp_with_time_zone.sql

```diff
@@ -15,5 +16,6 @@ ALTER TABLE moj_media DROP COLUMN c_reporting_restrictions;
 ALTER TABLE moj_media ALTER r_version_label TYPE CHARACTER VARYING(32);
 
 CREATE SEQUENCE IF NOT EXISTS notification_seq;
-CREATE SEQUENCE IF NOT EXISTS audio_request_seq;
+CREATE SEQUENCE IF NOT EXISTS media_request_seq;
 CREATE SEQUENCE IF NOT EXISTS moj_med_seq;
+CREATE SEQUENCE IF NOT EXISTS moj_hearing_media_ae_seq;
```

Comment:
Looks like the sequence is to be called `moj_hma_seq` in v22

https://cgigbr.sharepoint.com/teams/PRJ00013693/Shared%20Documents/Forms/All%20Documents.aspx?id=%2Fteams%2FPRJ00013693%2FShared%20Documents%2FGeneral%2FDesigns%2FBarry%2Fmodernised%5Fdarts%5Fschema%5Fv22%2Esql&parent=%2Fteams%2FPRJ00013693%2FShared%20Documents%2FGeneral%2FDesigns%2FBarry